package-lock.json

```diff
@@ -7,7 +7,7 @@
     "@babel/cli": {
       "version": "7.2.3",
       "resolved": "https://registry.npmjs.org/@babel/cli/-/cli-7.2.3.tgz",
-      "integrity": "sha512-bfna97nmJV6nDJhXNPeEfxyMjWnt6+IjUAaDPiYRTBlm8L41n8nvw6UAqUCbvpFfU246gHPxW7sfWwqtF4FcYA==",
```

Comment:
What version of npm are you using? I believe outdated npm versions use sha1 hash

Reply:
I'using version 8.11.3 and package-lock.json I've added to gitignore.